src/main/kotlin/uk/gov/dwp/dataworks/services/ExistingUserServiceCheck.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.dwp.dataworks.services
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import software.amazon.awssdk.services.ecs.EcsClient
+
+@Service
+class ExistingUserServiceCheck{
+    @Autowired
+    private lateinit var ecsDescribeServicesCall: EcsDescribeServicesCall
+    val configService = ConfigurationService()
+
+    fun check(userName: String, ecsClusterName: String):Boolean{
+        val ecsClient = EcsClient.builder().region(configService.awsRegion).build()
+
+        for (i in ecsDescribeServicesCall.servicesResponse(ecsClient, ecsClusterName).services()){
+            if(i.serviceName()=="${userName}-ui-service" && i.status()=="ACTIVE"){
```

Comment:
I feel like it would be better to include the service filter in the request to AWS as per the docs here? https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.describe_services
I think we can clean up this code a bit then, possibly removing the need for a loop entirely